lib/WorseReflection/Core/DocBlock/DocBlock.php

```diff
@@ -11,6 +11,8 @@
 
 interface DocBlock
 {
+    /** public function types(): Types; */
```

Comment:
I wasn't sure if it's okay to introduce new method in interface, so I've put this as comment for now